packages/odyssey-react-mui/src/labs/SideNav/SideNavItemContent.tsx

```diff
@@ -46,26 +49,33 @@ export const SideNavListItemContainer = styled("li", {
   "&:last-child": {
     marginBottom: odysseyDesignTokens.Spacing2,
   },
-  "& a": {
+  "& a, & div[role='button']": {
     display: "flex",
     alignItems: "center",
     width: "100%",
-    minHeight: "48px",
-    padding: `${odysseyDesignTokens.Spacing3} ${odysseyDesignTokens.Spacing4}`,
+    minHeight: isCompact ? "37px" : "48px",
+    padding: isCompact
+      ? `${odysseyDesignTokens.Spacing0} ${odysseyDesignTokens.Spacing4}`
+      : `${odysseyDesignTokens.Spacing3} ${odysseyDesignTokens.Spacing4}`,
     color: `${odysseyDesignTokens.TypographyColorHeading} !important`,
-    pointerEvents: isDisabled ? "none" : "auto",
   },
-  "& a:hover": {
+  "& a:hover, & div[role='button']:hover": {
     textDecoration: "none",
+    color: `${odysseyDesignTokens.TypographyColorAction} !important`,
     cursor: isDisabled ? "default" : "pointer",
     backgroundColor: !isDisabled ? odysseyDesignTokens.HueNeutral50 : "inherit",
   },
-  "& a:focus-visible": {
+  "& a:focus-visible, & div[role='button']:focus-visible": {
     outlineOffset: 0,
-    borderRadius: 0,
+    color: `${odysseyDesignTokens.TypographyColorAction} !important`,
```

Comment:
Yeah, I wanna work with you to figure out a better selector than `a` and `div[role="button"]`. This is gonna be problematic in the future.

Reply:
Refactored to not use selectors and moved the logic to within the component